src/packages/errors/index.ts

```diff
@@ -0,0 +1,20 @@
+import { setTags } from '@sentry/react';
+import React, { useEffect } from 'react';
+
+import { useAuth } from 'context/AuthContext';
+
+export const SentryEventTagger: React.FC = () => {
+  const { accountAddress, chainId } = useAuth();
+
+  // Tags Sentry events with the connected address and the chain ID being used
+  useEffect(() => {
+    setTags({ accountAddress, chainId });
```

Comment:
For the account address, I think we should use the `setUser` method from Sentry instead of `setTags` and use the account address as user ID, as Sentry as specific features around user identity.